augurpy/estimators.py

```diff
@@ -4,17 +4,16 @@
 from sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
 from sklearn.linear_model import LogisticRegression
 from switchlang import switch
-from typing_extensions import TypedDict
 
 
 @dataclass
-class Params(TypedDict, total=False):
+class Params:
     """Type signature for random forest and logistic regression parameters."""
 
-    n_estimators: int
-    max_depth: int
-    max_features: Union[Literal["auto"], Literal["log2"], Literal["sqrt"], int, float]
-    penalty: Union[Literal["l1"], Literal["l2"], Literal["elasticnet"], Literal["none"]]
+    n_estimators: int = 100
+    max_depth: Union[int, None] = None
```

Comment:
Isn't Union[int, None] basically Optional[int]?

Reply:
Yes! Ill change it to optional